packages/mock-account-servicing-lib/src/WebhookEventType.ts

```diff
@@ -0,0 +1,11 @@
+export enum WebhookEventType {
```

Comment:
we can also just throw this in `types.ts`

Reply:
moved